.github/workflows/codecov-nightly.yml

```diff
@@ -35,11 +31,9 @@ jobs:
           )
         shell: Rscript {0}
       - name: Show testthat output
-        if: always()
         run: |
-          ## --------------------------------------------------------------------
-          find ${{ runner.temp }}/package -name 'testthat.Rout*' -exec cat '{}' \; || true
-        shell: bash
+          cat ${{ runner.temp }}/package/testthat.Rout*
+        shell: cmd
```

Comment:
according to the [documentaion](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell) bash should work under windows as well (Git bash is used).
But somehow id did not. So trying cmd instead.

Reply:
Okay, I am wondering if this does not come from the runner.temp ? I got slash/backslash issues testing on my fork within the call of `RUNNER_TEMP` and the error message seemed to indicate a path that misses a slash.

Reply:
Somehow it comes from the combination of `runner.temp` and Windows.
I think it's a bug of GitHub. 
If it will not work with the cmd as well - I'm out of ideas.